include/mesh/xdr_io.h

```diff
@@ -161,6 +161,23 @@ class XdrIO : public MeshInput<MeshBase>,
   const std::string & polynomial_level_file_name() const { return _p_level_file; }
   std::string &       polynomial_level_file_name()       { return _p_level_file; }
 
+  /**
+   * @return true if the current file has an XDR/XDA version that
+   * matches or exceeds 0.9.2.
+   */
+  bool exceeds_version_0_9_2() const;
+
```

Comment:
Hmm... now that we're putting my stupid variable name into an API, perhaps we should make it less stupid?  "meets_or_exceeds_version_" is quite a mouthful, but accuracy is more important than brevity.  "version_at_least_"?